app/components/add-contributor/component.js

```diff
@@ -1,22 +1,27 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 import { and, notEmpty } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 import { isPresent } from '@ember/utils';
+import { isAddress } from 'web3-utils';
 
 export default Component.extend({
 
   kredits: service(),
 
   attributes: null,
 
-  // TODO: add proper address validation
-  isValidAccount: notEmpty('account'),
+  isValidAccount: computed('ethAddress', function() {
+    return isAddress(this.ethAddress);
```

Comment:
why is this now `ethAddress` and was `account` before?

Reply:
Typo. Thanks.